src/main/java/core/basesyntax/model/OperationFinderImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.model;
```

Comment:
Model is class that holds some properties.
This class looks more like util or service.
If you make method name as parseOperationFromInput - then it will perfectly fit your ParsingService and may be moved there